src/node_modules/dbpedia/ontology.mjs

```diff
@@ -0,0 +1,45 @@
+import { promises as fs } from 'fs';
```

Comment:
as `loadOntology` is only used by `getClasses` which is unused, I'd remove this file